rules/do-you-know-that-people-misunderstand-sarcasm-in-email/rule.md

```diff
@@ -1,7 +1,7 @@
 ---
 type: rule
 archivedreason: 
-title: Do you know that people misunderstand sarcasm in email?
+title: Do you know that people misunderstand sarcasm in writing?
```

Comment:
I agree with Penny, this is a good change but "Do you know that people can misunderstand sarcasm in text?" is even better.
```suggestion
title: Do you know that people misunderstand sarcasm in text?
```